src/components/InnerPopupWrapper/InnerPopupWrapper.types.ts

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react';
+
+export type InnerPopupWrapperProps = {
+  steps: number;
+  popupTitle?: string;
+  currentStep: number;
+  children: ReactNode;
```

Comment:
is it actually the correct way to type children? Also heard about the usage of React.FC<Props> that somehow has an impact on children. Any hints appreciated.

Reply:
I think that `React.ReactNode` is the best way here. You can read something about children typing here https://frontlive.pl/blog/react-children-i-typescript